Formula/awscli.rb

```diff
@@ -15,6 +15,7 @@ class Awscli < Formula
     sha256               mojave:        "9fa69138cad8cd539c851b1e95099eacfae8c3d933639ecbbfaaa5cff75a2e3a"
   end
 
+  depends_on "cmake" => :build
```

Comment:
this is now needed for the added dependency [awscrt](https://github.com/awslabs/aws-crt-python): https://github.com/aws/aws-cli/commit/016a27a8f6290d43658b1514f7391c96449d7e84